package.json

```diff
@@ -40,7 +40,7 @@
     "datatables": "1.10.18",
     "datatables.net-fixedcolumns": "3.2.6",
     "edx-proctoring-proctortrack": "git+https://git@github.com/anupdhabarde/edx-proctoring-proctortrack.git#f0fa9edbd16aa5af5a41ac309d2609e529ea8732",
-    "edx-ui-toolkit": "1.5.4",
+    "edx-ui-toolkit": "github:openedx/edx-ui-toolkit",
```

Comment:
Are we not publishing new versions of this automatically?  Sounds like an issue in an of itself.